docs/_data/navigation.yml

```diff
@@ -14,12 +14,8 @@
 primary:
   - text: Home
     href: /
-  - text: Open Source Policies
-    links:
-      - text: CrowdStrike Engineering
-        href: /open-source-policy/#engineering
-      - text: All Other Departments
-        href: /open-source-policy/#everyone-else
+  - text: Open Source Policy
+    href: /open-source-policy
```

Comment:
Neutral on this. 

So that we don't forget how to create the nested menu, can we comment out the old code instead of remove it?

Reply:
There's actually a bunch of commented examples in `navigation.yml` on nested menus already, so don't think it would be valuable to keep this.

Reply:
Got it. Thanks!